pipeline/cron-pipeline.groovy

```diff
@@ -54,21 +54,28 @@ node(nodeName) {
                 returnStdout: true,
                 script: "find ${defaultFileDir} -name RHCEPH-*.yaml -mtime -7 -ls"
             )
+            if (!updatedFiles) {
+                println "Recipe file updates are not available."
+                currentBuild.result = "ABORTED"
```

Comment:
Can we include a return statement or error?

Reply:
added error